javascript-loops/main.js

```diff
@@ -0,0 +1,68 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen() {
+  var number = [];
+  var currentNumber = 1;
+  while (currentNumber < 10) {
+    number.push(currentNumber);
+    currentNumber++;
+  }
+  return number;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  while (currentNumber < 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
```

Comment:
`getEvenNumbersToTwenty` should include `20`:

---

![image](https://user-images.githubusercontent.com/16143771/152256978-e5515d9d-8c89-4314-a888-d6da18327968.png)

---

![image](https://user-images.githubusercontent.com/16143771/152257014-a0cf4270-dd11-4037-90a0-750c09805841.png)